src/app/main/service/route-guards/auth-page-guard.service.ts

```diff
@@ -21,24 +21,31 @@ export class AuthPageGuardService implements CanActivate {
     state: RouterStateSnapshot
   ): Observable<boolean | UrlTree> | Promise<boolean | UrlTree> | boolean | UrlTree {
     if (!this.isLoggedIn) {
-      this.openSingInWindow();
-      return of(false);
+      if (this.localStorageService.getUbsRegistration() === 'true') {
+        this.openSingInSingUpWindow('sign-up');
+        return of(false);
+      } else if (this.localStorageService.getUbsRegistration() === 'false') {
+        this.openSingInSingUpWindow('sign-in');
+        return of(false);
+      }
```

Comment:
You can change this condition to 'if ' if it only accepts 'true' or 'false'
Example:
```
if(this.localStorageService.getUbsRegistration()){
    this.openSingInSingUpWindow('sign-up');
    return of(false);
}
this.openSingInSingUpWindow('sign-in');
return of(false);
```